lib/exchange.js

```diff
@@ -44,32 +51,68 @@ function exchange(name, type, options) {
   var ready = false;
   var connection, channel;
   var publishing = 0;
-  var replyQueue = queue({ exclusive: true });
+  var options = extend({}, DEFAULT_EXCHANGE_OPTIONS, options)
+  var replyQueue = !options.noReply && queue({ exclusive: true });
   var pendingReplies = {};
 
-  var emitter = _.extend(new EventEmitter(), {
+  var emitter = extend(new EventEmitter(), {
     name: name,
     type: type,
-    options: _.extend({}, DEFAULT_EXCHANGE_OPTIONS, options),
+    options: extend({}, DEFAULT_EXCHANGE_OPTIONS, options),
     queue: createQueue,
     connect: connect,
-    publish: publish
+    publish: publish,
+    rpcClient: rpcClient,
+    rpcServer: rpcServer
   });
 
   return emitter;
 
+  function rpcClient(key, msg, options, cb) {
```

Comment:
How is this related to "Add noReply option to exchange and enable passing of options paramaters during initialization" ?